doc/api/events.markdown

```diff
@@ -10,7 +10,8 @@ is opened. All objects which emit events are instances of `events.EventEmitter`.
 You can access this module by doing: `require("events");`
 
 Typically, event names are represented by a camel-cased string, however,
-there aren't any strict restrictions on that, as any string will be accepted.
+there aren't any strict restrictions on that, as any string or symbol will be
+accepted.
```

Comment:
Perhaps "any valid property key" is the most correct thing to put here? That encompasses everything that's actually supported and doesn't encourage *or* discourage the use of Symbols.